tests/test_fixtures/upgrade_helper/test_basic_project_upgrade_expected_stdout.fixture

```diff
@@ -61,11 +61,29 @@ A log detailing the upgrade can be found here:
 
 UpgradeHelperV13 will upgrade your project to be compatible with Great Expectations 0.13.x.
 
-Good news! No special upgrade steps are required to bring your project up to date.
```

Comment:
Why is this text being changed?  Especially surprising is the removal of the Y/n prompt -- we always need to ask the user whether or not they would like to proceed.

Reply:
This is updated because of the change to `default_checkpoints_exist`. With the second constraint removed, `default_checkpoints_exist` now evaluates to true for this test case, which means the CLI updater deems upgrade advised. 

The Y/n prompt is still there, just lower down.